src/Promitor.Integrations.Sinks.OpenTelemetry/Configuration/OpenTelemetryCollectorSinkConfiguration.cs

```diff
@@ -1,7 +1,11 @@
-﻿namespace Promitor.Integrations.Sinks.OpenTelemetry.Configuration
+﻿using OpenTelemetry.Exporter;
+
+namespace Promitor.Integrations.Sinks.OpenTelemetry.Configuration
 {
     public class OpenTelemetryCollectorSinkConfiguration
     {
+        public OtlpExportProtocol Protocol { get; set; } = default(OtlpExportProtocol);
         public string CollectorUri { get; set; }
+        //Todo: Headers
```

Comment:
Do we need this? We might want to remove this

Reply:
removing